content/blog/esc-new-onboarding/index.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "New Pulumi ESC Onboarding and OIDC Provider Experience"
+date: 2025-10-17T16:00:00-03:00
+draft: false
+meta_desc: Explore Pulumi ESC’s new onboarding and simplified OIDC setup for AWS, Azure & GCP. Securely manage environments with ease.
+meta_image: meta.png
+authors:
+    - pablo-terradillos
+    - sean-yeh
+tags:
+    - esc
+schema_type: auto
+---
+
+Pulumi ESC (Environments, Secrets, and Configuration) provides centralized secrets management and configuration orchestration across your infrastructure and applications. Today, we’re excited to introduce a redesigned onboarding experience and a simpler, automated way to set up Pulumi ESC as an OpenID Connect (OIDC) provider.
+
+<!--more-->
+
+## A new and refreshed onboarding experience
+
+Pulumi ESC helps you manage configuration for Pulumi programs, handle secret rotation, and centralize secrets stored across multiple providers.
+
+We’ve now started simplifying these workflows from the moment you create a new Environment. The redesigned onboarding experience helps you get the most out of ESC. From the very moment you’re starting fresh or cloning existing environments.
```

Comment:
i think the last sentence `From the very moment you’re starting fresh or cloning existing environments.` sounds a bit strange. probably can remove